_tour/basics.md

```diff
@@ -54,13 +51,13 @@ println(x) // 2
 Named results, such as `x` here, are called values. Referencing
 a value does not re-compute it.
 
-Values cannot be re-assigned.
+Values cannot be re-assigned:
 
 ```tut:fail
 x = 3 // This does not compile.
 ```
 
-Types of values can be inferred, but you can also explicitly state the type, like this:
+The type of a value can be omitted and inferred by [type inference](https://docs.scala-lang.org/tour/type-inference.html), or it can be explicitly stated:
```

Comment:
Being inferred by inference is kind of redundant.  Maybe say something like, "When no type is specified, [the type is inferred](...).  The type can also be explicitly specifed:"

Reply:
I take your point but I was aiming this at newcomers to Scala who might want to know what it is inferred by.

Reply:
They can figure it out easily enough with a link and a suggestive phrase like "type is inferred", no?

Reply:
Sorry, I was on mobile when I read that so I didn't see the hyperlink you add to "the type is inferred". Sure, that makes sense.